app/src/main/java/io/inai/android_sample_integration/drop_in/PresentCheckoutFragment.kt

```diff
@@ -0,0 +1,144 @@
+package io.inai.android_sample_integration.drop_in
+
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import android.view.View
+import io.inai.android_sample_integration.BuildConfig
+import io.inai.android_sample_integration.Config
+import io.inai.android_sample_integration.R
+import io.inai.android_sample_integration.helpers.NetworkRequestHandler
+import io.inai.android_sample_integration.helpers.json
+import io.inai.android_sample_integration.helpers.showAlert
+import io.inai.android_sdk.*
+import kotlinx.android.synthetic.main.fragment_present_checkout.*
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+
+class PresentCheckoutFragment : Fragment(R.layout.fragment_present_checkout), InaiCheckoutDelegate {
+
+    private val inaiBackendOrdersUrl: String = BuildConfig.InaiBaseUrl + "/orders"
```

Comment:
Can we rename this to `baseUrl` without `Inai` prefix? Also can we make this change across all places?

Reply:
Done